openwisp_utils/admin_theme/templates/admin/base_site.html

```diff
@@ -45,6 +45,20 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">{{ site_header }}</a></h1>
     <a href="{{ item.url }}"
        class="{{ item.class }}">{{ item.label }}</a>
     {% endfor %}
+    {% for group in openwisp_menu_groups %}
+        {% if  group.sub_items %}
+            <div class="ow-group-icon {{group.icon}}">{{group.label}}</div>
+            <div>
+            {% for item in group.sub_items %}
+            <a href="{{ item.url }}"
+               class="ow-menu-link"><i class="bi-alarm"></i><span class="{{item.icon}}"></span>{{ item.label }}</a>
```

Comment:
I would recommend to use `<i>` tag for presenting the icons. The `<span>` part could be removed in my opinion.

Also rework this changes according indentation and consistent formatting of template variables (for example `{{item.icon}}` should be `{{ item.icon }}`)

Reply:
Fixed indentation issue.
>Also rework this changes according indentation and consistent formatting of template variables (for example {{item.icon}} should be {{ item.icon }})

We can change it here https://github.com/openwisp/openwisp-utils/pull/185

Thanks @kkreitmair Its ready now but CI is not working